pkg/cmd/pipelineresource/describe.go

```diff
@@ -36,6 +36,9 @@ import (
 const templ = `{{decorate "bold" "Name"}}:	{{ .PipelineResource.Name }}
 {{decorate "bold" "Namespace"}}:	{{ .PipelineResource.Namespace }}
 {{decorate "bold" "PipelineResource Type"}}:	{{ .PipelineResource.Spec.Type }}
+{{- if ne .PipelineResource.Spec.Description "" }}
```

Comment:
Can a unit test be added for `tkn pr describe`?

Reply:
I missed adding builder for this in the pipeline

Reply:
For the time being, you could potentially just add something in our unit tests that would mimic what you're adding in pipeline. Then it could be replaced once that change goes in.

Reply:
done, thanks for review